src/diffpy/srxplanar/loadimage.py

```diff
@@ -66,31 +67,43 @@ def flipImage(self, pic):
         return pic
 
     def loadImage(self, filename):
-        """Load image file, if failed (for example loading an incomplete
-        file), then it will keep trying loading file for 5s.
+        """Load image file using pathlib. If loading fails (e.g.
```

Comment:
the use of pathlib is not relevant so don't have it in the docstring.  You don't mention that you use numpy for arrays in the docstring, for instance.